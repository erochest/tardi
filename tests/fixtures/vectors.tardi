use: std/vectors

<vector>

<vector>
7 over push
8 over push
9 over push

{ 1 2 3 4 5 }
dup pop
over pop
