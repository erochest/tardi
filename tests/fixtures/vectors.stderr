{ }
{ 7 8 9 }
{ 1 2 3 }
5
4
1
2
3
4
{ 1 2 7 4 }
{ 14 6 7 8 }
{ 9 10 11 1 }
0
1
2
6
1
4
{ 10 9 8 7 6 5 4 3 2 1 }
10
5
2
#f
#t
#f
#t
#f
#t
#f
#f
#f
0
#f
4
#f
{ }
{ 0 1 2 3 }
{ 8 9 }
{ 3 4 5 6 }
{ 5 6 7 }
"a, b, c"
"1 2 3"
""
{ }
{ 1 }
{ 1 2 3 }
{ 0 1 2 3 4 5 6 7 8 9 }
{ }
{ 1 2 3 }
{ 2 4 6 8 }
2
4
6
8
10
30
